manticore/platforms/linux.py

```diff
@@ -2318,6 +2318,12 @@ def _transform_write_data(self, data):
 
     #Dispatchers...
 
+    def sys_exit_group(self, error_code):
+        if issymbolic(error_code):
+            logger.debug("Solving for a symbolic process status code")
+            raise ConcretizeArgument(self, 0)
```

Comment:
this causes manticore to fork for me due to minmax. i think we should just solve for 1

```
2017-12-22 15:06:39,410: [94091] m.p.platform:DEBUG: sys_read(0x0, 0x7ffffffffd74, 0x4) -> 4
2017-12-22 15:06:39,645: [94091] m.p.linux:DEBUG: Solving for a symbolic process status code
2017-12-22 15:06:39,724: [94091] m.c.executor:INFO: Forking, about to store. (policy: MINMAX, values: 0x0, 0xffffffffffffffff)
2017-12-22 15:06:39,774: [94091] m.c.executor:INFO: load state 2
2017-12-22 15:06:39,782: [94091] m.p.linux:DEBUG: Solving for a symbolic process status code
2017-12-22 15:06:39,941: [94091] m.manticore:INFO: Generated testcase No. 0 - Program finished with exit status: -1
2017-12-22 15:06:39,957: [94091] m.c.executor:INFO: load state 1
2017-12-22 15:06:39,963: [94091] m.p.linux:DEBUG: Solving for a symbolic process status code
2017-12-22 15:06:40,123: [94091] m.manticore:INFO: Generated testcase No. 1 - Program finished with exit status: 0
2017-12-22 15:06:40,134: [94091] m.c.plugin:INFO: Coverage: 2572 different instructions executed
2017-12-22 15:06:40,137: [94091] m.manticore:INFO: Results in /mnt/hgfs/code/manticore/examples/linux/mcore_UGSMq2
2017-12-22 15:06:40,137: [94091] m.manticore:INFO: Total time: 4.89450192451

```

Reply:
I also think we should try to incorporate the `(*)` output convention here to signal that a concrete value came from a symbolic value. something like concretizing, and not calling super?